tutorials/group_level_analysis/plot_group_level_with_individual_clusters.py

```diff
@@ -0,0 +1,81 @@
+"""
+Group level analysis from individual clusters
+====================
```

Comment:
```suggestion
Group-level analysis from individual clusters
=============================================
```

Match the number of characters.